docs/installation.md

```diff
@@ -202,35 +237,198 @@ freqtrade new-config --config config.json
 
 > *To edit the config please refer to [Bot Configuration](configuration.md).*
 
-#### 6. Run the Bot
+#### Run the Bot
 
 If this is the first time you run the bot, ensure you are running it in Dry-run `"dry_run": true,` otherwise it will start to buy and sell coins.
 
 ```bash
 freqtrade trade -c config.json
 ```
 
+#### Problem?
+
+Check if your virtual environment is activated
+
+```bash
+source ./.env/bin/activate
+```
+
```

Comment:
The title of this suggests that this is the only problem that can occur - which it definitely isn't.

This is one problem (the easiest to solve) - but there's many others - by naming this section "problem?" you suggest that this will always help...

Reply:
OK